docs/package-types/esm-view.md

```diff
@@ -16,8 +16,6 @@ following essential differences:
 1. All their external `import`s are rewritten to point to an
    [ESM CDN](../esm-views/esm-cdn.md).
 
-1. They don't allow the user to specify a custom `index.html`
```

Comment:
Do you also need to update the table on the main page? It currently has a cross emoji against `Custom index / assets`

Reply:
Good catch - done!